src/Runner.Listener/JobDispatcher.cs

```diff
@@ -55,6 +57,11 @@ public sealed class JobDispatcher : RunnerService, IJobDispatcher
 
         private TaskCompletionSource<bool> _runOnceJobCompleted = new TaskCompletionSource<bool>();
 
+        public event EventHandler<EventArgs> JobStarted;
+        public event EventHandler<EventArgs> JobCompleted;
+
+        private ITerminal _term;
```

Comment:
you might want to update line 363 to use the new `_term` you added, it was getting the terminal inline.

Reply:
Was going to remove that.. just added for my debugging.